src/WebJobs.Script/FileProvisioning/PowerShell/PowerShellFileProvisioner.cs

```diff
@@ -3,14 +3,29 @@
 
 using System;
 using System.IO;
+using System.Net.Http;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
+using System.Xml;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.Script.FileProvisioning.PowerShell
 {
     internal class PowerShellFileProvisioner : IFuncAppFileProvisioner
     {
+        private const string AzModuleName = "Az";
+        private const string PowerShellGalleryFindPackagesByIdUri = "https://www.powershellgallery.com/api/v2/FindPackagesById()?id=";
+
+        private const string ProfilePs1FileName = "profile.ps1";
+        private const string RequirementsPsd1FileName = "requirements.psd1";
+
+        private readonly ILogger _logger;
+
+        public PowerShellFileProvisioner(ILogger logger)
+        {
+            _logger = logger;
```

Comment:
Create logger here. Creating logger will show up with the source 'PowerShellFileProvisioner ' which is more useful

Reply:
Fixed.

Reply:
This needs to be PowerShellFileProvisioner(ILoggerFactory loggerFactory)